core/domain/question_domain.py

```diff
@@ -229,3 +237,64 @@ def update_question_data(self, question_data):
             question_data: dict. A dict representing the question data.
         """
         self.question_data = question_data
+
+    def add_skill(self, skill_name, user_id):
```

Comment:
All methods here need tests, too.

Reply:
I think this method is tested in test_update_skill_methods.

Reply:
Ah, yes, you're right. That said, could we break up that test so that each method gets its own set of tests? That would help to ensure we're testing each method thoroughly.

Reply:
Updated.